examples/computation_graph_02.ml

```diff
@@ -61,16 +61,16 @@ let visualise_lstm () =
   let yt', _ = Graph.(init network; forward network xt) in
   let loss = A.(Maths.((cross_entropy yt yt') / (pack_flt (Mat.row_num yt |> float_of_int)))) in
   let _, adj0 = Graph.(backward network loss) in
-
-  let s0 = G.nodes_to_dot [| loss |> A.unpack_elt |> G.elt_to_node |] in
+  let inputs = [| xt |> A.unpack_arr |> G.arr_to_node |] in
+  let s0_outputs = [| loss |> A.unpack_elt |> G.elt_to_node |] in
+  let s0 = G.make_graph inputs s0_outputs "mnist_loss" |> G.graph_to_dot in
   Owl_io.write_file "cgraph_04_lstm_loss.dot" s0;
   (* Sys.command "dot -Tpdf -Gnslimit=1 cgraph_04_lstm_loss.dot -o cgraph_04_lstm_loss.pdf" |> ignore; *)
-
-  let s1 = adj0
+  let s1_outputs = adj0 
     |> Utils.Array.flatten
     |> Array.map (fun a -> A.unpack_arr a |> G.arr_to_node)
-    |> G.nodes_to_dot
   in
+  let s1 = G.make_graph inputs s1_outputs "mnist_loss" |> G.graph_to_dot in
   Owl_io.write_file "cgraph_04_lstm_grad.dot" s1
   (* Sys.command "dot -Tpdf -Gnslimit=1 cgraph_04_lstm_grad.dot -o cgraph_04_lstm_grad.pdf" |> ignore *)
```

Comment:
@pratapsingh1729 fyi, there is owlviz tool which can help you to visualise the computation graph: https://github.com/owlbarn/owl/blob/master/bin/dune